proto/backup.proto

```diff
@@ -118,7 +118,7 @@ message S3 {
     string prefix = 4;
     string storage_class = 5;
     // server side encryption
-    bool sse = 6;
+    string sse = 6;
```

Comment:
@overvenus does this `6` need to be changed to another tag?

Reply:
I think it's okay to keep `6` since the new `S3` is not used yet.  BTW, why does it need to be changed to `string`?

Reply:
It is a `string` instead of `bool`, please check detail in the [here](https://rclone.org/s3/#s3-server-side-encryption).